tests/conftest.py

```diff
@@ -99,16 +99,17 @@ def endpoint_fixture(
     port_forward_payload: list[dict[str, Any]],
     site_payload: list[dict[str, Any]],
     system_information_payload: list[dict[str, Any]],
+    traffic_rule_payload: list[dict[str, Any]],
     wlan_payload: list[dict[str, Any]],
 ) -> None:
     """Use fixtures to mock all endpoints."""
 
     def mock_get_request(
-        path: str, unifi_path: str, payload: list[dict[str, Any]]
+        path: str, unifi_path: str, payload: list[dict[str, Any]], v2: bool = False
     ) -> None:
         """Register HTTP response mock."""
         url = unifi_path if is_unifi_os else path
-        data = {"meta": {"rc": "OK"}, "data": payload}
+        data = payload if v2 else {"meta": {"rc": "OK"}, "data": payload}
```

Comment:
```suggestion
        data = payload if path.startswith("/v2") else {"meta": {"rc": "OK"}, "data": payload}
```
Not sure which is cleanest, your v2 option or just looking at the path

Reply:
For the v2 there are also 2 paths possible, depending if we are dealing with UnifiOS or not: `"/proxy/network/v2/api/site/{site}{self.path}"` and `"/v2/api/site/{site}{self.path}"`. 
I think your suggestion should then read `data = payload if "v2" in path else {"meta": {"rc": "OK"}, "data": payload}`
Both options work :wink: , but with the v2-boolean the responsibility for determining whether it is a v2 API or not is left to the user of the method. Therefore, this felt more natural for me.

Reply:
Well both path and unifi_path would be a v2 path right? That's why I'm only suggesting looking at the path string which starts with V2